### PR TITLE
more updates to Polish translation

### PR DESCRIPTION
**Pull request by agilob**

### CODE REVIEW

**Comment by tux3** [comment]
You didn't update the .qm, should I do it ?

**Comment by agilob** [comment]
wait a moment , one translation is incorrect.

File: translations/pl.ts [hunk 1]
```diff
@@ -410,7 +406,7 @@ Czy chcesz usunąć ten profil ?</translation>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="74"/>
         <source>Auto away after (0 to disable):</source>
         <translatorcomment>better translation? anyone?</translatorcomment>
-        <translation>Automatyczna zmiana statusu na &quot;Nieobecny&quot; po (0 by wyłączyć):</translation>
+        <translation>Zmiana statusu na &quot;zajęty&quot; po (0 by wyłączyć):</translation>
```
Comment (zetok) [inline translations/pl.ts]: Um, it doesn't change to "Zajęty" but to "Nieobecny":

**Comment by agilob** [comment]
I hope it will work @tux3

**Pull request closed without merging**